compute/instances/api-cli/migrating-instances.mdx

```diff
@@ -1,213 +1,111 @@
 ---
 meta:
-  title: Migrating Instances with the API
-  description: This page explains how to migrate Instances with Scaleway API
+  title: Changing the commercial type of an Instance with the API
+  description: This page explains how to change the commercial type of an Instance with the API
 content:
-  h1: Migrating Instances with the API
-  paragraph: This page explains how to migrate Instances with Scaleway API
-tags: migration migrate instance api
+  h1: Changing
+  paragraph: This page explains how to the commercial type of an Instance with the API
+tags: upgrade instance api
 dates:
-  validation: 2023-04-25
+  validation: 2023-06-29
   posted: 2021-05-26
 categories:
   - compute
 ---
 
-It is possible to automate the migration between two Instances by using the API.
+It is possible to change the commercial type of an Instance by using the API.
 
-To migrate an Instance via the API, the instance ID, a valid API Key and the users organization key are required.
+To migrate an Instance via the API, the Instance ID, and a valid API Key are required.
 
 <Macro id="iam-requirements" />
 
-1. Start by making a backup of your Instance. Replace `<REGION>` with the region of your server. This value can be `fr-par-1` for Instances located in Paris, for example.
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have [created an Instance](/compute/instances/how-to/create-an-instance)
+  - The Instance must be stopped for the migration to be possible
+  - Local storage constraints must be respected (For example, downgrading a DEV1-L instance with 80GB of local SSD storage to a DEV1-S instance is not possible because the DEV1-S offer only supports a maximum of 20GB. However, if the DEV1-L instance is only using block storage (`b_ssd`) without relying on local SSD storage, the downgrade can be performed without any issues.)
```

Comment:
Do we need this in the requirements ? Looks redundant with the section below

Reply:
I agree, it will be enough to have it in the warning below.